udata/tests/api/test_oembed_api.py

```diff
@@ -82,3 +100,113 @@ def test_oembeds_dataset_api_get_without_valid_id(self):
         self.assert400(response)
         self.assertEqual(json.loads(response.data)['message'],
                          'Unknown dataset ID.')
+
+    def test_oembeds_api_for_territory(self):
+        '''It should fetch a territory in the oembed format.'''
+        country = faker.country_code()
+        level = 'town'
+        zone = GeoZoneFactory(level='{0}/{1}'.format(country, level))
+        org = OrganizationFactory()
+
+        class TestDataset(TerritoryDataset):
```

Comment:
I can't, it needs to have reference on runtime organization ID :/
But I can factorize it